darts/models/forecasting_model.py

```diff
@@ -95,6 +96,194 @@ def _build_forecast_series(self,
 
         return TimeSeries.from_times_and_values(time_index, points_preds, self.training_series.freq())
 
+    def _backtest_sanity_checks(self, series: TimeSeries, start: pd.Timestamp, forcast_horizon: int):
+        """Perform sanity checks on backtest inputs.
+
+        Parameters
+        ----------
+        series
+            The univariate time series on which to backtest
+        start
+            The first prediction time, at which a prediction is computed for a future time
+        forcast_horizon
+            The forecast horizon for the point predictions
+
+        Raises
+        ------
+        ValueError
+            if an input doesn't pass sanity checks.
+        """
+        raise_if_not(start in series, '`start` timestamp is not in the `series`.', logger)
+        raise_if_not(start != series.end_time(), '`start` timestamp is the last timestamp of `series`', logger)
+        raise_if_not(forcast_horizon > 0, 'The provided forecasting horizon must be a positive integer.', logger)
+
+    def _backtest_model_specfic_sanity_checks(self, retrain: bool):
+        """Add model specific sanity check(s) on the backtest inputs.
+
+        Parameters
+        ----------
+        retrain
+            Whether to retrain the model for every prediction or not. Currently only `TorchForecastingModel`
+            instances as `model` argument support setting `retrain` to `False`.
+
+        Raises
+        ------
+        ValueError
+            if an input doesn't pass sanity checks.
+        """
+        raise_if(not retrain, "Only 'TorchForecastingModel' instances support the option 'retrain=False'.", logger)
+
+    def _backtest_build_fit_and_predict_kwargs(self,
+                                               target_indices: Optional[List[int]],
+                                               component_index: Optional[int],
```

Comment:
this doesn't seem to be used

Reply:
You mean because it is overriden? It is used as default and not rewritten in the multivariate case so used

Reply:
As we discussed, I was talking about `component_index` that is apparently not used in this method.